Task-1.c

```diff
@@ -0,0 +1,37 @@
+#include <stdio.h>
+#include <string.h>
+#include <locale.h>
+
+int main()
+{
+    setlocale(LC_ALL, "ru");// просто установка ру 

```

Comment:
```suggestion
    setlocale(LC_ALL, "ru"); // установка русской локали
```
Комментарии должны быть грамотно написанными. Ведь комментарий предназначен для того, чтобы его кто-то читал, ему будет удобнее и понятнее, если грамотно и в меру развернуто написать.